spec/requests/review_spec.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Reviews API', type: :request do
+  # initialize test data
+  let(:place) { create(:place) }
+  let!(:reviews) { create_list(:review, 10, place: place) }
```

Comment:
👏

Reply:
no tthat i am for this, but it might also be possible to do this:
`create(:place, reviews: create_list(:review, 3))`
then you wouldnt need to relate them to the previously created `place`

(you can try it just for fun)